encoding/ccf/simpletype_test.go

```diff
@@ -52,7 +52,9 @@ func TestTypeConversion(t *testing.T) {
 	}
 
 	for ty := interpreter.PrimitiveStaticType(1); ty < interpreter.PrimitiveStaticType_Count; ty++ {
-		if !ty.IsDefined() || ty.IsDeprecated() { //nolint:staticcheck
+		if !ty.IsDefined() ||
+			ty.IsDeprecated() || //nolint:staticcheck
+			ty == interpreter.PrimitiveStaticTypeStorable {
```

Comment:
I didn't add support for "exporting" storable static type, because it not something a user can write/use (only used in subtyping checks). But noticed we already support exporting `Any` static type, which is also not something user's can use in their code. Do we maybe want to support the storable-type in a similar way?

Reply:
If we're adding a new primitive type we should also add support for converting/exporting it. I'm not quite sure how this would be encountered, but we should handle it properly if it is.

Reply:
Added in https://github.com/onflow/cadence/pull/4335/commits/0c933655e25442d01e09eb04dc0cb44fb367c3fb